src/creatures/combat/condition.cpp

```diff
@@ -552,6 +587,47 @@ void ConditionAttributes::updateSkills(Player* player) {
 	}
 }
 
+void ConditionAttributes::updatePercentAbsorbs(const Creature* creature) {
+	for (uint8_t i = 0; i < COMBAT_COUNT; i++) {
+		auto value = getAbsorbPercentByIndex(i);
+		if (value == 0) {
+			continue;
+		}
+		setAbsorb(i, 100 - creature->getAbsorbPercent(indexToCombatType(i)) * value / 100);
```

Comment:
```suggestion
		setAbsorb(i, std::round((100 - creature->getAbsorbPercent(indexToCombatType(i))) * value / 100.f));
```
Without float division and rounding we truncate this down to zero almost all the time.
The parens are also necessary other wise we're doing 100-0 and the player is always immune to the element.

This isn't something easy to test here because it was added for concoctions but I'm working on the concoctions now and ran up against it.